pkg/executor/alisa.go

```diff
@@ -37,13 +38,25 @@ const (
 	entryFile    = "entry.py"
 )
 
+const (
+	// AlisaTaskTypePAI is PAI task in the Alisa task enumeration
+	AlisaTaskTypePAI = iota
+	// AlisaTaskTypePyODPS is PyODPS task in the Alisa task enumeration
+	AlisaTaskTypePyODPS
+)
+
 var reOSS = regexp.MustCompile(`oss://([^/]+).*host=([^&]+)`)
 
 type alisaExecutor struct {
 	*pythonExecutor
 }
 
-func (s *alisaExecutor) submitAlisaTask(submitCode, codeResourceURL, paramsResourceURL string) error {
+type alisaTaskContext struct {
+	codeResourceURL   string
+	paramsResourceURL string
+}
+
+func (s *alisaExecutor) submitAlisaTask(taskType int, submitCode string, args string, context *alisaTaskContext) error {
```

Comment:
In general, `args` should be a string list as in C/C++:
``` c++
void main(int argc, char ** args)  {
 ...
}
```

Do you want to make `args string` to `args []string` ?